src/games/ambipoms-tossups.ts

```diff
@@ -27,6 +29,8 @@ class AmbipomsTossups extends QuestionAndAnswer {
 	updateHintTime: number = 5 * 1000;
 
 	static loadData(): void {
+		// eslint-disable-next-line max-len
+		data["Pokemon Locations"] = Dex.getLocations().filter(x => x.length > 3);data["Pokemon Locations"] = Dex.getLocations().filter(x => x.length > 3);
```

Comment:
Looks like this was accidentally duplicated.

Reply:
Right, I will make sure it is resolved

Reply:
Removed the duplicate, thanks